docs/src/udp.rst

```diff
@@ -181,6 +181,23 @@ API
 
     :returns: 0 on success, or an error code < 0 on failure.
 
+.. c:function:: int uv_udp_set_source_membership(uv_udp_t* handle, const char* multicast_addr, const char* interface_addr, const char* source_addr, uv_membership membership)
+
+    Set membership for a source-specific multicast group. Currently implemented only for IPv4.
```

Comment:
What's the story for IPv6? I tried to look it up, but other than some Cisco IOS docs and some FreeBSD stuff there is not much I can find...